lib/node_modules/@stdlib/blas/base/wasm/cscal/docs/types/index.d.ts

```diff
@@ -465,21 +443,11 @@ interface Routine extends ModuleWrapper {
 *
 * // Perform operation:
 * cscal.main( x.length, z, x, 1 );
-*
-* var v = x.get( 0 );
-* // returns <Complex64>
-*
-* var re = realf( v );
-* // returns -2.0
-*
-* var im = imagf( v );
-* // returns 6.0
+* // x => <Complex64Array>[ -2.0, 6.0, -2.0, 14.0, -2.0, 22.0]
```

Comment:
```suggestion
* // x => <Complex64Array>[ -2.0, 6.0, -2.0, 14.0, -2.0, 22.0 ]
```

Use consistent spacing.